sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -583,6 +583,12 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
       .booleanConf
       .createWithDefault(true)
 
+  val CHUNKED_SUBPAGE_READER = conf("spark.rapids.sql.reader.chunked.subPage")
+      .doc("Enable a chunked reader where possible for reading data that is smaller " +
+          "than the typical row group/page limit. Currently this only works for parquet.")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
We should have a followup issue to turn this on in 24.04.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/10334